app/credentials/header.go

```diff
@@ -0,0 +1,6 @@
+package credentials
+
+const (
+	AuthorizationHeader = "Authorization"
+	AuthorizationBearer = "Bearer"
```

Comment:
`BearerAuthorization` seems clearer to me, wdyt?  

(feel free to ignore)

Reply:
Hmm I'm not sure it is much clearer, but also agree that `AuthorizationBearer` is not great. What about `BearerTokenType` or `TokenTypeBearer`?

Reply:
`BearerTokenType` sgtm!